package.json

```diff
@@ -8,6 +8,12 @@
     "start": "yarn build && yarn workspace example-app start",
     "bundle": "yarn build && yarn workspace example-app bundle",
     "build": "lerna run build",
+    "clean": "yarn run clean:cli && yarn run clean:app && yarn run clean:core && yarn run clean:storybook && yarn clean:root",
+    "clean:root": "rimraf node_modules/",
+    "clean:cli":"rimraf packages/cli/dist/ packages/cli/node_modules/",
+    "clean:app":"rimraf packages/app/node_modules/",
+    "clean:core": "rimraf packages/core/dist packages/core/node_modules",
+    "clean:storybook":"rimraf packages/storybook/node_modules",
```

Comment:
This would be easier to manage by leveraging `lerna run`, where the `clean` scripts are defined in each package. also, clean should probably be a command exposed by `backstage-cli`.

Reply:
I am following this issues #427 to add the clean script. @fastfrwrd do you think I need to add the script into backstage-cli?

Reply:
Thanks for looking into this! And yes, thinks have changed a bit since the issue opened and I think we need to add a `clean` command to the `backstage-cli`.

I think we should view the clean as something to do when debugging issues or developing the CLI, as in it should reset all caching mechanisms that are in place.  Currently that includes the `dist` dir in each package, as well as the build cache that by default is located in `node_modules/.cache/backstage-builds`. It's configurable though, and that happens [here](https://github.com/spotify/backstage/blob/84340367c497343ffd774a8ecefdb4ae474e4428/packages/cli/src/commands/build-cache/options.ts#L56).

I think we likely want the clean command for each package to remove it's own cache, so e.g. running `clean` inside `packages/core` would delete `packages/core/dist`, and `node_modules/.cache/backstage-builds/packages/core`.

With a clean command for each package we'll then implement the top-level clean using lerna run, as @fastfrwrd mentioned.